packages/terra-list/src/ListItem.jsx

```diff
@@ -102,10 +114,20 @@ const ListItem = ({
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
   }
+  if (isDraggable) {
+    attrSpread['aria-describedby'] = responseId;
+  }
 
   return (
     <li {...customProps} {...attrSpread} className={listItemClassNames} ref={refCallback}>
-      <div className={cx('item-fill')} key="item-fill">{children}</div>
+      <div className={cx('item-fill')} key="item-fill">
+        {children}
+        {(isDraggable) && (
+          <React.Fragment>
```

Comment:
Question: Why the fragment for a single item?

Reply:
Changes Done. Thanks